easybuild/easyconfigs/s/SQuIRE/SQuIRE-0.9.9.9a-beta-foss-2023a-Python-2.7.18.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'SQuIRE'
+version = '0.9.9.9a-beta'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/wyang17/SQuIRE'
+description = """Software for Quantifying Interspersed Repeat Expression"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '2.7.18'),
+    ('pyfaidx', '0.7.1', versionsuffix),
+]
+
+use_pip = True
+
+# Remove `urllib3` requirement as it is not used in the code.
+preinstallopts = "sed -i \"s/,'urllib3\\[secure\\]'//g\" setup.py && "
+
+exts_list = [
+    (name, version, {
+        'source_urls': ['https://github.com/wyang17/SQuIRE/archive/'],
+        'sources': ['v%(version)s.tar.gz'],
```

Comment:
@PetrKralCZ Can you update this to use the latest commit (and adjust the version accordingly, by including a datestamp that corresponds to that commit)?

There seem to be potentially important bug fixes that are not included in the tagged release, like https://github.com/wyang17/SQuIRE/commit/bfeb0356324866cbd20381705de6e12c2bc65f6d